cypress.json

```diff
@@ -0,0 +1,4 @@
+{
+  "baseUrl": "https://ui.circleci.com",
```

Comment:
Will this work when we run cypress in CI?

Reply:
When you run in CI you can configure an override to the default baseURL. Alternatively I can remove the baseURL and add in both script commands the override manually so there is no confusing. In addition when looking int Nav code for the spun up enviroment it was configured for ui.circleci.com so I did not find an issue with making it the default as the enviroment in CI was the same